source/common/network/udp_listener_impl.h

```diff
@@ -60,7 +60,6 @@ class UdpListenerImpl : public BaseListenerImpl,
   void disableEvent();
 
   TimeSource& time_source_;
-  Event::FileEventPtr file_event_;
```

Comment:
Shouldn't this be removed from the tcp listener as well?

Reply:
resolving here, so we can have one thread on the tcp listener vs udp listener and if we can always use REUSE_PORT